### PR TITLE
Add Content Template section to docs

### PR DESCRIPTION
**Pull request by mateoholman**
This PR will add a section for Content Templates to the Getting Started -> Back Office section of the docs, per Issue #989 
I'm open to any suggested edits / changes!

### CODE REVIEW

File: Getting-Started/Backoffice/Content-Templates/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# Content Templates
+Content Templates allow you to create a blueprint for new content nodes based on an existing node.
```
Comment (dampee) [inline Getting-Started/Backoffice/Content-Templates/index.md]: Can we replace "you" with a role?  E.g. content editor?

File: Getting-Started/Backoffice/Content-Templates/index.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Create - Method 1
+
+Select the 'Content' menu: 
+
+![Content Menu](../images/01-Content-Menu.png)
```
Comment (dampee) [inline Getting-Started/Backoffice/Content-Templates/index.md]: Going one level up with images won't work on the OUR documentation website as that according to @jmayntzhusen .  We need to have images in a subfolder of the current file.

File: Getting-Started/Backoffice/Content-Templates/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Click the 'Create' button and if the creation was successful, you will see a success notification:
+
+![Create Button](../images/04-Save-Content-Template.png)
+
+The new content template will be created in 'Content Templates' node of the settings tree:
+
+![New Content Template](../images/05-Find-Content-Template.png)
+
+## *Create - Method 2*
```
Comment (dampee) [inline Getting-Started/Backoffice/Content-Templates/index.md]: having bold and a header is a little overkill, right?  Just remove the star's

File: Getting-Started/Backoffice/Content-Templates/index.md (hunk 1 above)
Comment (mateoholman) [reply Getting-Started/Backoffice/Content-Templates/index.md]: Hi @dampee I replaced this instance of you with the suggested 'content editor'. Did you want to replace all instances of 'you'?

File: Getting-Started/Backoffice/Content-Templates/index.md (hunk 2 above)
Comment (mateoholman) [reply Getting-Started/Backoffice/Content-Templates/index.md]: No problem, done!

**Comment by mateoholman** [comment]
@dampee I setup a profile on our.umbraco.com with the username mateoholman@gmail.com.
I think I addressed all your comments, but let me know if you see anything else!

**Comment by dampee** [comment]
That's almost ace.  There was a small typo, but I fixed it already.  :-)

Thanks a lot!

**Comment by nul800sebastiaan** [comment]
Hi there @mateoholman,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**